### ISSUE TITLE
newline to enable comment-ending SQL

### ISSUE THREAD

**Issue by RobbertDM** [innoverio/vscode-dbt-power-user#361]
### Describe the feature

https://github.com/innoverio/vscode-dbt-power-user/blob/bf18ec4fc580bf952122e9e767d2008711be8c98/package.json#L83

what do you say we add a `\n` here?
```
 "default": "select * from ({query}\n) as query limit {limit}" 
```

Tested and works for previewing SQL :) 
Then you can have a comment on the last line of your SQL file

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by mdesmet** [comment]
If it works I'm fine adding it.

**Comment by mdesmet** [comment]
Fixed in 0.12.2
